addons/account/views/account_view.xml

```diff
@@ -404,6 +405,30 @@
            res_model="account.move.line"
            src_model="account.journal"/>
 
+        <!-- Tags -->
+        <record id="account_tag_view_form" model="ir.ui.view">
+            <field name="name">Tags</field>
+            <field name="model">account.account.tag</field>
+            <field name="arch" type="xml">
+                <form string="Tags">
+                    <group>
+                        <field name="name"/>
+                    </group>
+                </form>
+            </field>
+        </record>
+        <record id="account_tag_action" model="ir.actions.act_window">
+            <field name="name">Tags</field>
+            <field name="res_model">account.account.tag</field>
+            <field name="view_type">form</field>
+            <field name="help" type="html">
+              <p class="oe_view_nocontent_create">
+                Click to add a new tag.
+              </p>
+            </field>
+        </record>
+        <menuitem action="account_tag_action" id="menu_action_account_tag_form" parent="account_account_menu" groups="base.group_no_one"/>
```

Comment:
Menu name should be in **model_name_menu** pattern  and also place **id** before all other attributes.